java-api-examples/Makefile

```diff
@@ -56,17 +71,32 @@ runmic:
 
 		java -cp ./lib/sherpaonnx.jar:build $(RUNJFLAGS)  DecodeMic
 
+runsrv:

```

Comment:
Could you document the message format/protocol for the communication?

Reply:
ok